packages/@o3r/dynamic-content/README.md

```diff
@@ -38,19 +38,19 @@ A pipe to be used in your component templates:
 
 ```html
 <img src="{{'assets-otter/imgs/logo.png' | dynamicContent}}" /> or
-<img [src]="'assets-otter/imgs/logo.png' | dynamicContent" />
+<img ngSrc="{{'assets-otter/imgs/logo.png' | dynamicContent}}" fill/>
```

Comment:
why using `ngSrc` ?
`fill` is useful for background image but not recommend for others images as it will cause layout shift
So I would not recommend we put that in our recommendation

Reply:
After discussion, ngSrc is recommended over [src] for performances: https://angular.io/guide/image-directive